ocs_ci/deployment/deployment.py

```diff
@@ -173,6 +173,13 @@ def deploy_cluster(self, log_cli_level="DEBUG"):
             setup_netsplit(
                 tmp_path, master_zones, worker_zones, x_addr_list, arbiter_zone
             )
+        if config.ENV_DATA["deploy_acm_hub_cluster"]:
```

Comment:
I am bit unclear here, how do we know on which OCP cluster we want ACM to be deployed ?

Reply:
+1

Reply:
i have added a condition where we deploy acm over 4.9 or later cluster